src/systems/label/Label.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include <limits>
+#include <string>
+
+#include <ignition/plugin/Register.hh>
+#include "ignition/gazebo/components/Label.hh"
+#include "ignition/gazebo/components/World.hh"
+#include "ignition/gazebo/components/Visual.hh"
+#include "ignition/gazebo/components/Link.hh"
+#include "ignition/gazebo/EntityComponentManager.hh"
+#include "Label.hh"
```

Comment:
```suggestion
 */
 
#include "Label.hh"

#include <string>

#include <ignition/plugin/Register.hh>

#include "ignition/gazebo/EntityComponentManager.hh"
#include "ignition/gazebo/components/Label.hh"
#include "ignition/gazebo/components/Link.hh"
#include "ignition/gazebo/components/Visual.hh"
```

See https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

It also seems like you never use the `World` component, so I don't think you need to include the header for it. I also don't think you're using anything from `<limits>`, so I removed that include statement as well.